contrib/libsass.spec

```diff
@@ -1,7 +1,7 @@
 Name:           libsass
-Version:        3.1.0
+Version:        %{version}
```

Comment:
Actually it is meant to be replaced manually (the file should just serve as a template). I just thought it is worth a try that way, but I have no idea if this actually works or has any benefits.